.github/workflows/spelling.yml

```diff
@@ -0,0 +1,18 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Check Website Links"
```

Comment:
Wrong workflow name

Reply:
Renamed.

In my workflows I use this naming system.
- workflow name: what is being checked, usually a noun
- job name: the category of the tool/s used, e.g. `static_analysis`
- step name: what is being run, verb + noun